core/commands/quick_stage.py

```diff
@@ -12,7 +12,8 @@
 CLEAN_WORKING_DIR = "Nothing to commit, working directory clean."
 ADD_ALL_UNSTAGED_FILES = " ?  All unstaged files"
 ADD_ALL_FILES = " +  All files"
-
+STAGED = "--- {} files are staged for commit. ---"
+COMMIT = "    git: commit"
 
```

Comment:
Do you have a list of which errors you care about vs ignore so I can configure this a little tighter?  Right now I'm getting flags for lines longer than 79, which seems to be  pretty common issue in this source. :)